File: Basic/code/problem_2.c
/*
Input principal amount, rate and time and calculate S.I.
*/


// Solved by glitchy-coder-exe

#include<stdio.h>
#include<conio.h>
int main(){
    float p,r,t,si;
    clrscr();
    printf("Enter the principal amount(in rupees), rate(in percent) and time(in years): \n");
    scanf("%f\n%f\n%f",&p,&r,&t);
    si=p*r*t/100;
    printf("The simple interest is Rs. %f",si);
    getch();
    return 0;
}

// Solved by Krishna7472
#include<stdio.h>
int main(){
    float p,r,t,si;
    printf("Enter the principal amount(in rupees),rate(in percent) and time(in years): \n");
    scanf("%f\n%f\n%f",&p,&r,&t);
    si=(p*r*t)/100;
    printf("The simple interest is Rs. %.3f",si);
    return 0;
}

//Solved By ParthThaker007

#include<stdio.h>
int main(){
    int p,r,t;
    float si;
    cin>>p>>r>>t;
    si=(p*r*t)/100;
    cout<<"Simple Interest is : "<<si;
    return 0;
}
    

File: Basic/code/problem_2.java
import java.util.Scanner;  

public class problem_2{
  
  public static void main(String args[]){  
int p, r, t, SI;  
    
Scanner sc = new Scanner(System.in);  
    
System.out.print("Enter Principal Amount: ");  
p = sc.nextInt();  
    
System.out.print("Enter Principal Rate: ");  
r = sc.nextInt();  
    
System.out.print("Enter Time in Years: ");  
t = sc.nextInt(); 
    
SI = SI(p, r, t);  
System.out.println("Simple Interest: " + SI);  
}  
//method that calculates the sum  
public static int SI(int a, int b, int c)  
{  
int SI = (a*b*c)/100;  
return SI;  
}  
}
